library/x509_crt.c

```diff
@@ -180,6 +180,9 @@ static int x509_profile_check_md_alg( const mbedtls_x509_crt_profile *profile,
 static int x509_profile_check_pk_alg( const mbedtls_x509_crt_profile *profile,
                                       mbedtls_pk_type_t pk_alg )
 {
+    if (pk_alg == MBEDTLS_PK_NONE)
```

Comment:
Thanks a lot for your contribution! I support this fix, but could you switch to the Mbed TLS coding style here? (No space after `if`, but spaces before and after brackets.

Reply:
Of course :-)
Is there a tool to check coding style before submitting ?

Reply:
@catenacyber Unfortunately, I am not aware of any.